<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>throughputEdit</class>
 <widget class="QWidget" name="throughputEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>throughput config Edit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/test/throughput</normaloff>:/test/throughput</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_type">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>iperf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>chariot</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_desc"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tw_iperf">
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
         <item row="5" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>bitrate(-b):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="chk_reverse">
           <property name="text">
            <string>reverse(-R)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="sb_parallel">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>window size(-w):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Test Duration(-t, sec):</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QComboBox" name="cb_unit_windowsize">
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>130</height>
            </size>
           </property>
           <property name="title">
            <string>manager:</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>server ip:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_mIPserver">
              <property name="toolTip">
               <string>iperf server will be run at here</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>client ip:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_mIPclient">
              <property name="toolTip">
               <string>iperf client will be run at here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="sb_bitrate">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(0 for unlimited)&lt;/p&gt;&lt;p&gt; (default 1 Mbit/sec for UDP, unlimited for TCP)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="sb_windowsize"/>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_18">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IPv4: 536 octets (= 576 - 20 - 20) &lt;/p&gt;&lt;p&gt;IPv6: 1220 octets (= 1280 - 40 - 20). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Max segment size(-M,--set-mss):</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QComboBox" name="cb_fmtreport">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>k</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>g</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>t</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>T</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>format to report(-f):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="sb_duration">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>864000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="sb_dscp">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-1: default value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>63</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Parallel(-P):</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>IP DSCP bits(--dscp):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="chk_bidir">
           <property name="text">
            <string>bidirectional(--bidir)</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="sb_omit">
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QSpinBox" name="sb_mss">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0: default value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>window size unit:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>omit(-O, sec):</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QComboBox" name="cb_unit_bitrate">
           <property name="currentText">
            <string>K</string>
           </property>
           <item>
            <property name="text">
             <string>K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>130</height>
            </size>
           </property>
           <property name="title">
            <string>test:</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>server ip(-c, --client):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_server"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>client ip(-B, --bind):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_client"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>bitrate unit:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>iperf version:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_version">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Port(-p, --port)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_port">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>5201</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>protocal:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_protocal">
              <item>
               <property name="text">
                <string>TCP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UDP</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qiperf.qrc"/>
 </resources>
 <connections/>
</ui>
